deploy/crds/eunomia_v1alpha1_gitopsconfig_crd.yaml

```diff
@@ -104,7 +104,7 @@ spec:
                   type: string
               type: object
             triggers:
-              description: Triggers is an array of triggers that will lanuch this
+              description: Triggers is an array of triggers that will launch this
```

Comment:
Please fix in `pkg/apis/eunomia/v1alpha1/gitopsconfig_types.go` instead and run `make generate`

Reply:
Yeah ... `make generate` is a better way to do this instead of using `operator-sdk` directly. :-)